app/src/main/java/com/example/coopertest/TestActivity.kt

```diff
@@ -168,12 +169,14 @@ class TestActivity : AppCompatActivity(), OnMapReadyCallback {
     }
 
     private fun currentDistanceString(): String {
-        return "%.0fm".format(currentDistanceMeters)
+        if (!miles) {
+            return "%.0f m".format(currentDistanceMeters)
+        } else {
+            return "%.0f M".format(currentDistanceMeters/1609.344)
```

Comment:
I think we should show it the same way, either km / miles, or meters / yards. I prefer meters / yards